feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -129,6 +129,26 @@ func (s *RelayRouter) getSessionProxyClient(
 	return protos.NewCentralSessionControllerClient(conn), ctx, cancel, nil
 }
 
+//N7_proxy service for 5G
+func (s *RelayRouter) getN7ProxyClient(
```

Comment:
`getN7N40ProxyClient`

Maybe we can rename the whole service to N740... .but that may be confusing. We can think about it